front/cypress/e2e/admin/pages_and_menu/create_update_and_view_custom_page.cy.ts

```diff
@@ -0,0 +1,154 @@
+import { randomString } from '../../../support/commands';
+
+describe('Admin: create, update, and view custom page', () => {
+  before(() => {
+    cy.setAdminLoginCookie();
+    cy.visit('/admin/pages-menu/');
+    cy.acceptCookies();
+  });
+
+  it('creates a custom page successfully', () => {
+    cy.intercept('POST', '**/static_pages/**').as('createCustomPage');
```

Comment:
Why do we need to intercept it? The same question about `getCustomPage`. How do we use these interceptions?

Reply:
True, I created the three interceptions but only use `@updateCustomPage` later in the code. Basically it's a way of waiting for a call to successfully complete. In this case it's to double check that clicking a toggle actually makes a call to the backend.

I can remove the other unused two though

Reply:
> I can remove the other unused two though

Maybe I would prefer to do it to avoid confusion. But not a big deal, it's fine as it is now too.

Reply:
I'll get rid of them when I make the other changes, I created those up front thinking I would use them and didn't remove them when it turned out they are unnecessary.